### DISCUSSION TITLE
Entity Brain Storm

### DISCUSSION THREAD

**Discussion by cloudyyoung** [cloudyyoung/course-enrolment-utility-frontend#1] [General]
Existing ones (**strong entities in bold**):

- [x] **Faculty**
- [x] **Department**
- [x] **Program** (Computer Science, Chemistry, etc)
- [x] **Course**
- [x] Section (LEC 01, TUT 01; an iteration of a course in a specific semester)
- [x] **Staff**

**Comment by cloudyyoung** [comment]
- [ ] Course Field

![image](https://user-images.githubusercontent.com/20573623/137071302-075005c5-58be-4279-9f43-7e084124f530.png)
[https://www.ucalgary.ca/pubs/calendar/current/sc-4-3-1.html](https://www.ucalgary.ca/pubs/calendar/current/sc-4-3-1.html)

**Comment by cloudyyoung** [comment]
- [ ] Major (subclass of Program) Recommendation
- [ ] Minor (subclass of Program) Recommendation
- [ ] Regular Major (subclass of Major) Recommendation
- [ ] Honor Major (subclass of Major) Recommendation
- [ ] Concentration (weak entity relates to Major) Recommendation

![image](https://user-images.githubusercontent.com/20573623/137252259-d116b7ac-01ea-4231-8d45-b6010d372eb6.png)

**Comment by Zhezhee** [comment]
concentration

**Comment by cloudyyoung** [comment]
- [x] **Semester** (Fall 2021, Winter 2022)
- [ ] **_University_**
- [ ] **_Club_**

**Comment by cloudyyoung** [comment]
- [x] **User**
- [x] Student (subclass of user)
- [x] Admin (subclass of user)
